src/Field.cpp

```diff
@@ -1109,14 +1094,26 @@ omnmmsgFieldPayload_getAsString       (const msgFieldPayload   field,
     {
         const void* result   = NULL;
         mama_size_t dataSize = 0;
-        mama_size_t s        = 0;
-
         status = omnmmsgFieldPayload_getOpaque (field, &result, &dataSize);
-
-        for (s = 0; s < dataSize; s++)
-        {
-            snprintf (buffer, len, "%#x ", ((char*) result)[s]);
+        #if 1
```

Comment:
No `#if 1` please - either macro the whole thing to its own METHOD() or else take it out entirely.

Reply:
Wasn't sure if you were keen on having the actual contents printed, so I left both in for now.  (I didn't want to lose the fix for the original code, which was broken).

Personally I find the size to be plenty good enough, esp. since we deal with rather large blob fields.  If that's OK I can take out the printing of contents.

Reply:
Yeah lets take the else section out - keep it clean,